### PR TITLE
Documenting GSCTL_CAFILE and GSCTL_CAPATH

### PR DESCRIPTION
**Pull request by marians**
Related to https://github.com/giantswarm/gsctl/pull/97

This will add info on two new environment variables to https://docs.giantswarm.io/reference/gsctl/ and https://docs.giantswarm.io/reference/gsctl/global-options/#switching-environments

Pinging @rossf7 since you were already involved and the rest for knowledge sharing.

### CODE REVIEW

File: content/reference/gsctl/global-options.md [hunk 1]
```diff
@@ -44,4 +44,10 @@ $ inhousegsctl list clusters
 ...
 ```
 
+If your API endpoint is making use of a self-signed certificate, you can throw the environment variable `GSCTL_CAFILE` into the mix and let it point to the CA certificate. Example:
```
Comment (rossf7) [inline content/reference/gsctl/global-options.md]: <3 the use of "into the mix" one of my favourite expressions.

File: content/reference/gsctl/index.md [hunk 2]
```diff
@@ -91,6 +91,8 @@ sudo cp gsctl-{{% gsctl_version %}}-linux-amd64/gsctl /usr/local/bin/</code></pr
 
 The following environment variables can be used to affect some behaviour:
 
+- `GSCTL_CAFILE`: If your Giant Swarm API endpoint uses a certificate signed by an authority not known to your operating system, this variable can be set to the path of a custom CA (certificate authority) bundle. A CA bundle is a text file containing one or several CA certificates in PEM format.
```
Comment (rossf7) [inline content/reference/gsctl/index.md]: I think "one or more" is better.

File: content/reference/gsctl/index.md [hunk 3]
```diff
@@ -91,6 +91,8 @@ sudo cp gsctl-{{% gsctl_version %}}-linux-amd64/gsctl /usr/local/bin/</code></pr
...
+- `GSCTL_CAPATH`: Similar to `GSCTL_CAFILE`, but `GSCTL_CAPATH` is expected to point to a directory containing one or several PEM files.
```
Comment (rossf7) [inline content/reference/gsctl/index.md]: Same again

File: content/reference/gsctl/global-options.md (hunk 1 above)
Comment (marians) [inline content/reference/gsctl/global-options.md]: My pleasure ;-)

**Pull request merged**